java/src/jmri/jmrix/AbstractThrottleManager.java

```diff
@@ -606,11 +596,14 @@ public boolean addressStillRequired(LocoAddress la) {
 
     @Override
     public void releaseThrottle(DccThrottle t, ThrottleListener l) {
+        log.debug("AbstractThrottleManager.releaseThrottle: {}, {}", t.toString(), l.toString());
```

Comment:
We want to avoid using `toString()` in logging statements, since the String gets generated before determining if the logging statement should be written. The fastest way to (not) log in this case is `log.debug("AbstractThrottleManager.releaseThrottle: {}, {}", t, l);`.

Reply:
fixed in subsequent commit by removing the .toString() invocations, as requested..